core/src/test/java/org/mule/processor/IdempotentRedeliveryPolicyTestCase.java

```diff
@@ -98,6 +100,9 @@ public ObjectStore answer(InvocationOnMock invocation) throws Throwable
             }
         });
         when(event.getMessage()).thenReturn(message);
+
+        IdempotentRedeliveryPolicyTestCase.serializer = SerializationTestUtils.getJavaSerializerWithMockContext();
```

Comment:
This test already has a mulecontext, should not use that instance to configure the serializer?